examples/terraform/aws/output.tf

```diff
@@ -85,7 +85,7 @@ output "kubeone_workers" {
       replicas = var.initial_machinedeployment_replicas
       providerSpec = {
         annotations = {
-          "k8c.io/operating-system-profile" = var.ami_filters[var.os].osp_name
+          "k8c.io/operating-system-profile" = var.initial_machinedeployment_operating_system_profile == "" ? var.ami_filters[var.os].osp_name : var.initial_machinedeployment_operating_system_profile
```

Comment:
Please use local `initial_machinedeployment_operating_system_profile` that is initialized like here.

Reply:
Updated.